src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,10 @@
+@mixin rule-fonts ($fnt-wight, $fnt-size, $fnt-line-height) {
+  font-weight: $fnt-wight;
+  font-size: $fnt-size;
+  line-height: $fnt-line-height;
+}
+
+@mixin rule-flex {

```

Comment:
Not mistake, you can also use this mixin
`@mixin flex($direction: row, $align: initial, $justify: initial) {
  display: flex;
  flex-direction: $direction;
  align-items: $align;
  justify-content: $justify;
}`